bitnami/fluent-bit/values.yaml

```diff
@@ -0,0 +1,681 @@
+  ##
+  logLevel: info
+  ## @param config.service [string] Defines the global behaviour of the Fluent Bit engine.
+  ##
+  service: |
+    [SERVICE]
+        Flush        {{ .Values.config.flush }}
+        Daemon       Off
+        LogLevel     {{ .Values.config.logLevel }}
+        Config_Watch On
+        HTTP_Server  On
+        HTTP_Listen  0.0.0.0
+        HTTP_Port {{ .Values.containerPorts.http }}
+  ## @param config.inputs [string] Defines the source from where Fluent Bit can collect data
+  ## https://docs.fluentbit.io/manual/pipeline/inputs
+  ##
+  inputs: |
+    [INPUT]
+        Name cpu
+
+    [INPUT]
+        Name systemd
+        Tag host.*
+        Systemd_Filter _SYSTEMD_UNIT=kubelet.service
+        Read_From_Tail On
+
+  ## @param config.filters [string] Set of plugins that can be used to filter, modify, or enrich log data that is processed by Fluent Bit.
+  ## https://docs.fluentbit.io/manual/pipeline/filters
+  ##
+  filters: |
+    [FILTER]
+        Name                kubernetes
+        Match               kube.*
+        Kube_URL            https://kubernetes.default.svc:443
+        Kube_CA_File        /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        Kube_Token_File     /var/run/secrets/kubernetes.io/serviceaccount/token
+        Kube_Tag_Prefix     kube.var.log.containers.
+        Merge_Log           Off
+        Merge_Log_Key       log_processed
+        K8S-Logging.Parser  On
+        K8S-Logging.Exclude Off
+  ## @param config.outputs [string] Outputs to send the collected data to different destinations
+  ## https://docs.fluentbit.io/manual/pipeline/outputs
+  ##
+  outputs: |
+    [OUTPUT]
+        Name  stdout
+        Match *
+  ## @param config.upstream This configuration is deprecated, please use `extraFiles` instead.
+  ## https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/upstream-servers
+  ##
+  upstream: {}
+  ## @param config.customParsers [string] Custom parsers
+  ## https://docs.fluentbit.io/manual/pipeline/parsers
+  ##
+  customParsers: |
+    [PARSER]
+        Name docker_no_time
+        Format json
+        Time_Keep Off
+        Time_Key time
+        Time_Format %Y-%m-%dT%H:%M:%S.%L
+  ## @param config.extraFiles Extra config files
+  ## This allows adding more files with arbitary filenames to /fluent-bit/etc by providing key/value pairs.
+  ## The key becomes the filename, the value becomes the file content.
+  ##
+  extraFiles: {}
+## Role Based Access
+## Ref: https://kubernetes.io/docs/admin/authorization/rbac/
+## @param rbac.create Create Role and RoleBinding
+## @param rbac.nodeAccess RBAC node access
+##
+rbac:
+  create: false
+  nodeAccess: false
+
+## @section Autoscaling
+##
+autoscaling:
+  vpa:
+    ## @param autoscaling.vpa.enabled Enable VPA
+    ##
+    enabled: false
+    ## @param autoscaling.vpa.annotations Annotations for VPA resource
+    ##
+    annotations: {}
```

Comment:
I don't see any reference to this value

Reply:
Fixed [a103df7](https://github.com/bitnami/charts/pull/15453/commits/a103df7c57edb633207f15507d3bf1f252f81bc2)